SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -38,6 +39,9 @@ void FSpatialGDKEditorModule::StartupModule()
 	ExtensionManager->RegisterExtension<FGridLBStrategyEditorExtension>();
 	SpatialGDKEditorInstance = MakeShareable(new FSpatialGDKEditor());
 	CommandLineArgsManager->Init();
+
+	FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
```

Comment:
I thought the editor module gets loaded before the services module, is that not the case?

Reply:
It seems that the Services module is loaded first and then Editor and Toolbar module

Reply:
Ahh I see, `SpatialGDK` and `SpatialGDKServices` modules have `PreDefault` loading stage and the rest are `Default` (except `SpatialGDKTests` which is `PreLoadingScreen`).

Reply:
Could you add a comment here that we rely on the module loading phase here? Just in case that changes and this starts breaking the next person would hopefully know what to do.